src/main/java/game/GameEngine.java

```diff
@@ -40,8 +41,13 @@ public static void main(String[] args) {
                         createNewMap(fileName);
                     }
                     loadMap(path, map);
-                    endGame();
-                } else {
+                    // endGame();
+                }
+                else if (commandArgs.length == 1 && "showmap".equals(commandArgs[0])) {
+                    Map Map;
```

Comment:
Don't see any use of `Map Map`. This line should be removed.

Reply:
Done